crates/holochain/src/conductor/conductor/tests.rs

```diff
@@ -524,6 +524,7 @@ async fn make_signing_call(client: &mut WebsocketSender, cell: &SweetCell) -> Ap
 /// not seem to be an issue, therefore I'm not putting the effort into fixing it
 /// right now.
 #[tokio::test(flavor = "multi_thread")]
+// TODO is this worth keeping?
 #[ignore = "we need a better mock keystore in order to implement this test"]
 #[allow(unreachable_code, unused_variables, unused_mut)]
 async fn test_signing_error_during_genesis_doesnt_bork_interfaces() {
```

Comment:
I don't know about this one...seems like another ignored test that leaves a clue for someone later on. It even describes exactly what's needed to write the test. Could be useful?